### PR TITLE
Add Python 3.10 to GHA

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

### CODE REVIEW

File: .github/workflows/test-python-package.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, "3.10"]
```
Comment (taylorfturner) [inline .github/workflows/test-python-package.yml:16]: @gliptak 
```suggestion
        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

**Review by taylorfturner — COMMENTED** [review]
just a small update and then will approve

File: .github/workflows/test-python-package.yml (hunk 1 above)
Comment (JGSweets) [reply .github/workflows/test-python-package.yml:16]: https://github.com/actions/setup-python/issues/160

Comment (JGSweets) [reply .github/workflows/test-python-package.yml:16]: Looks like it might be fixed though at this point.

**Review by taylorfturner — APPROVED** [review]

**Comment by taylorfturner** [comment]
Thank you for both of the pull requests, @gliptak!

**Review by micdavis — APPROVED** [review]

**Pull request merged**